src/_includes/feature_lists/cloud.njk

```diff
@@ -2,47 +2,75 @@
         }, {
+            id: "audit-log",
             label: "Audit Log",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>Keep track of everything going on in your Node-RED instances and FlowForge.</p><p>Audit Logs provide details on what actions have taken place, when they happened, and who did them.</p>"
         }, {
+            id: "team-config",
             label: "Team-Based Configuraton",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>Manage all of your Node-RED projects in FlowForge Teams, control access and Team-level settings in a central place to reduce operational overheads.</p>"
         }, {
+            id: "rbac",
             label: "Role-Based Access Control",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>FlowForge's intuitive team management tooling makes it easy for you to control whom has access to what.</p>"
         }, {
+            id: "device-mgmt",
             label: "Edge Device Management",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>Manage and control thousands of devices in FlowForge. Define your flow in one central place, then one-click deploy remotely to the edge, all from within FlowForge.</p>"
         }, {
+            id: "version-history",
             label: "Version History",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>FlowForge Snapshots provide capability to save backups of your flows. Revert back to past versions, and use those Snapshots to deploy out to remote devices.</p>"
         }, {
+            id: "persistent-file",
             label: "Persistent File Storage",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>Read & write to files in your Project flows. These will persist beyond container and project restarts and upgrades.</p>"
         }, {
+            id: "sso",
             label: "Single Sign-On (SSO)",
-            values: [null, 'check']
+            values: [null, 'check'],
+            info: "<p>FlowForge can be configured to run with your own SSO provider. Use the single username/password combination for your organisation, and prevent the need for users to create bespoke FlowForge login credentials.</p>"
         }, {
+            id: "project-nodes",
             label: "Inter Project Communication",
-            values: [null, 'check']
+            values: [null, 'check'],
+            info: "<p><a href=\"https://github.com/flowforge/flowforge-nr-project-nodes\" target=\"blank\">FlowForge Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowForge Projects by default.</p>"
```

Comment:
@robmarcer Do we have proper documentation for these notes? Otherwise we really should. Can you create an issue?